src/modules/page/admin/content.php

```diff
@@ -220,72 +222,42 @@
 $nv_Lang->setGlobal('title_suggest_max', $nv_Lang->getGlobal('length_suggest_max', 65));
 $nv_Lang->setGlobal('description_suggest_max', $nv_Lang->getGlobal('length_suggest_max', 160));
 
-$xtpl = new XTemplate('content.tpl', NV_ROOTDIR . '/themes/' . $global_config['module_theme'] . '/modules/' . $module_file);
-$xtpl->assign('LANG', \NukeViet\Core\Language::$lang_module);
-$xtpl->assign('GLANG', \NukeViet\Core\Language::$lang_global);
-$xtpl->assign('FORM_ACTION', $action);
-$xtpl->assign('UPLOADS_DIR_USER', NV_UPLOADS_DIR . '/' . $module_upload);
-$xtpl->assign('DATA', $row);
-$xtpl->assign('BODYTEXT', $row['bodytext']);
-$xtpl->assign('SOCIALBUTTON', ($row['socialbutton']) ? ' checked="checked"' : '');
-$xtpl->assign('HOST_POST', ($row['hot_post']) ? ' checked="checked"' : '');
-$xtpl->assign('ISCOPY', $copy);
-
+// Xử lý activecomm_array trước khi assign DATA
+$activecomm = array_map('intval', explode(',', $row['activecomm']));
+$row['activecomm_array'] = $activecomm;
+
+$tpl = new \NukeViet\Template\NVSmarty();
+$tpl->setTemplateDir(get_module_tpl_dir('content.tpl'));
+$tpl->assign('LANG', $nv_Lang);
+$tpl->assign('FORM_ACTION', $action);
```

Comment:
@huydepzai121 thống nhất ưu tiên các thao tác nối chuỗi, if, while, for.. cơ bản trong smarty hơn trong php. Cái gì nó làm được, không quá phức tạp thì sử dụng Smarty nhé em. Ở đây nối [kiểu như này](https://github.com/nukeviet/nukeviet/blob/nukeviet5.0/src/themes/admin_future/modules/news/content.tpl#L26) là được không cần sinh thêm nhiều biến đơn như extemplate